Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/FileSystemInvocation.java

```diff
@@ -0,0 +1,109 @@
+package com.lantanagroup.link.measureeval;
+
+import ca.uhn.fhir.context.FhirContext;
+import com.lantanagroup.link.measureeval.services.MeasureEvaluator;
+import org.apache.commons.io.FileUtils;
+import org.hl7.fhir.r4.model.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class is used to invoke measure evaluation using arbitrary artifacts on the file system for ease of testing and debugging new measures.
+ * Format: `java -jar measureeval.jar -cp com.lantanagroup.link.measureeval.FileSystemInvocation "<measure-bundle-path>" "<patient-bundle-path>" "<start>" "<end>"`
```

Comment:
Good point. I missed this because I was only running it from within IntelliJ. I updated docs to use your sytnax above, and tested that it works from the command line after building the jar with `mvn install -pl measureeval -am`.